torch/csrc/jit/tensorexpr/CMakeLists.txt

```diff
@@ -36,22 +36,22 @@ include("${ASMJIT_DIR}/CMakeLists.txt")
 include_directories("${ASMJIT_DIR}/src")
 
 add_library(nnc
```

Comment:
Can you name this cmake library `tensorexpr` instead of `nnc`

Reply:
Let's do this in a follow-up. Also, it might become irrelevant once I start to move things from our cmake to PyTorch's cmake.